prescription/integrity.go

```diff
@@ -14,7 +14,6 @@ const (
 
 type IntegrityAttributes struct {
```

Comment:
[nit] Technically no need for `IntegrityAttributes` now and could just use `DataAttributes`.